### PR TITLE
Sets up twitch bot 

### LINKED ISSUE #17: Create Twitch Bot

### LINKED ISSUE #20: Create Twitch Commands
!like - adds a like for the current song. 
!song - lists the current playing song. 
!artist - displays social media info for the current playing artist

### PR DESCRIPTION
**Pull request by camcast3**
Twitch Bot configured to handle like, song and artist commands for twitch chat. 

Updated .env file most up to date usage.

Some refactoring done in the repo. Also left some future work within TODO's for better optimization.

### CODE REVIEW

File: bots/twitch_bot.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        load_dotenv()
+        postgres = postgresproxy.PostgresProxy(
+            user=os.getenv("POSTGRES_USER"), password=os.getenv("POSTGRES_PASSWORD"))
+        socials = await postgres.artist_info_command(current_artist)
+
+        await mpd.mpd_connection_close()
+        await postgres.postgres_connection_close()
+
+        return_message = f'''Artist Name: {current_artist} \n
+        YouTube: {socials[0]}'''
```
Comment (prosolis) [inline bots/twitch_bot.py]: What are your thoughts on changing this to 'Socials' or 'Artist Link' or something similar?

File: requirements.txt [hunk 2]
```diff
@@ -1,17 +1,24 @@
+aiohttp==3.8.5
+aiosignal==1.3.1
 astroid==2.15.5
+async-timeout==4.0.2
+attrs==23.1.0
 cachetools==3.1.1
-certifi==2023.7.22
```
Comment (prosolis) [inline requirements.txt:3]: Why did we regress here?  Google foolishness?

Comment (camcast3) [reply requirements.txt:3]: probably something I did on my end

File: bots/twitch_bot.py (hunk 1 above)
Comment (camcast3) [reply bots/twitch_bot.py]: the name or the the message itself?

File: requirements.txt (hunk 2 above)
Comment (camcast3) [reply requirements.txt:3]: fixed

**Review by prosolis — APPROVED** [review]

**Pull request merged**